src/components/RawQueryEditor.tsx

```diff
@@ -69,6 +69,17 @@ export const RawQueryEditor: React.FC<RawQueryEditorProps> = (props) => {
       signatureHelpTriggerCharacters: ['(', ')'],
       provideSignatureHelp: getSignatureHelp,
     });
+    monaco.languages['kusto']
```

Comment:
This should be returned if the caller of `handleEditorMount` is expecting a promise.

Reply:
not really needed to return a promise.